src/library/Cardano/CoinSelection/Fee.hs

```diff
@@ -9,36 +9,32 @@
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeFamilies #-}
+{-# OPTIONS_HADDOCK prune #-}
 
 -- |
 -- Copyright: © 2018-2020 IOHK
 -- License: Apache-2.0
 --
--- Provides the API of Coin Selection algorithm and Fee Calculation
--- This module contains the implementation of adjusting coin selection for a
--- fee.  The sender pays for the fee and additional inputs are picked randomly.
--- For more information refer to:
--- https://iohk.io/blog/self-organisation-in-coin-selection/
-
+-- Provides functionality for __adjusting__ coin selections in order to pay for
+-- transaction __fees__.
+--
 module Cardano.CoinSelection.Fee
     (
-      -- * Fee Calculation
+      -- * Fundamental Types
       Fee (..)
-    , calculateFee
-    , distributeFee
+    , FeeEstimator (..)
+    , DustThreshold (..)
 
       -- * Fee Adjustment
-    , FeeEstimator (..)
-    , FeeOptions (..)
-    , ErrAdjustForFee(..)
     , adjustForFee
-    , reduceChangeOutputs
+    , FeeOptions (..)
+    , FeeAdjustmentError (..)
 
-      -- * Dust Processing
-    , DustThreshold (..)
+      -- # Internal Functions
```

Comment:
This doesn't need the `#` does it?

Reply:
> This doesn't need the `#` does it?

Strictly speaking, it's not necessary.

I just put it there to create a "section title" that isn't picked up by Haddock.